src/SFA.DAS.FindEmploymentSchemes.Web/Views/CaseStudies/CaseStudyPage.cshtml

```diff
@@ -1,20 +1,39 @@
-﻿@model CaseStudyPage
+﻿@using SFA.DAS.FindEmploymentSchemes.Contentful.Model.Content
+@model CaseStudyPage
 
 @{
-    ViewData["Title"] = Model.Title.Substring(0, Math.Min(Model.Title.Length, 64));;
+    ViewData["Title"] = Model?.Title?.Substring(0, Math.Min(Model?.Title?.Length ?? 64, 64));
```

Comment:
If we introduce a new error view, we won't have to change the existing views (such as this one, page and scheme) to cater for missing mandatory fields, or have a custom PreviewErrors section, i.e. they can remain unchanged.

We'd compile a list of errors by checking the preview content in the controllers (or delegated to a service), then if there are any errors, we can display an error view, rather than the normal view.